### PR TITLE
Minor e2e fixes

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot-Parab <prajyot.parab2@ibm.com>

**What this PR does / why we need it**:
- Minor e2e fixes

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

/area provider/ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
Minor e2e fixes
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -203,7 +203,7 @@ test: generate fmt vet setup-envtest $(GOTESTSUM) ## Run tests
 GINKGO_FOCUS ?= Workload cluster creation
 GINKGO_NODES ?= 3
 GINKGO_NOCOLOR ?= false
-E2E_FLAVOR ?= powervs
+E2E_FLAVOR ?= md-remediation
```
Comment (mkumatag) [inline Makefile:206]: wondering if we can make it more explicit like `powervs-md-remediation` for better understanding

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Prajyot-Parab) [reply Makefile:206]: we can that will also need renaming of template file name. should we go ahead with that @mkumatag?

Comment (mkumatag) [reply Makefile:206]: I hope that's the file name we are talking about in the `tests` folder, if yes - go ahead..

Comment (Prajyot-Parab) [reply Makefile]: Fixed.

File: api/v1beta2/ibmvpccluster_types.go [hunk 2]
```diff
@@ -59,8 +59,8 @@ type IBMVPCClusterSpec struct {
 // VPCLoadBalancerSpec defines the desired state of an VPC load balancer.
 type VPCLoadBalancerSpec struct {
 	// Name sets the name of the VPC load balancer.
-	// +kubebuilder:validation:MaxLength:=64
-	// +kubebuilder:validation:Pattern=`^[A-Za-z0-9]([A-Za-z0-9]{0,31}|[-A-Za-z0-9]{0,30}[A-Za-z0-9])$`
+	// +kubebuilder:validation:MaxLength:=63
+	// +kubebuilder:validation:Pattern=`^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$`
```
Comment (Amulyam24) [inline api/v1beta2/ibmvpccluster_types.go:63]: @Prajyot-Parab, can you mention here on why this change is being made?

Comment (Prajyot-Parab) [reply api/v1beta2/ibmvpccluster_types.go:63]: @Amulyam24 The new pattern is inline with the one used by loadbalancer API ([https://cloud.ibm.com/apidocs/vpc-beta#create-load-balancer](https://cloud.ibm.com/apidocs/vpc-beta#create-load-balancer)), previous one seemed to have its limitation when accepting strings greater than 32 chars (that is fixed now).

**Review by Amulyam24 — APPROVED** [review]
/lgtm

**Review by mkumatag — APPROVED** [review]
